src/components/ArticleIndividual/ArticleIndividual.jsx

```diff
@@ -20,7 +22,26 @@ function ArticleIndividual() {
         setLoading(false);
       });
   }, []);
+  function handleVoteButton(event) {
+    // typeof is string, so make int
+    const updateVote = parseInt(event.target.value);
 
+    // make patch request
+    apiClient
+      .patch(`/api/articles/${article_id}`, {
+        inc_votes: updateVote,
+      })
+      .then((response) => {
+        // dont do anything with response, as OPTIMISTIC LOADING is used
+        setArticle((prevArticleItem) => ({
+          ...prevArticleItem,
+          votes: prevArticleItem.votes + updateVote,
+        }));
```

Comment:
So this is not optimistic, because you are doing the PATCH before the state is updated. 

You should update the votes first, then do the patch. Then it will be optimistic!